codecheck.py

```diff
@@ -46,7 +52,7 @@ def check_code():
             subprocess.run(
                 [
                     "cargo",
-                    "doc",
+                    "build",
```

Comment:
Should probably be `cargo check` instead for better performance, essentially the same as `cargo build` but doesn't actually compile it.